.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,17 @@
+## 🌴 PR 요약
+
+🌱 작업한 브랜치
+- feature/#
+
+## 🌱 PR Point
+
+## 📌 참고 사항
+<!-- 참고할 사항이 있다면 적어주세요. -->
+
+## 📸 스크린샷
+|기능|스크린샷|
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)